README.md

```diff
@@ -1,20 +1,22 @@
-# CADDY-RUN [![Build Status](https://travis-ci.org/lucaslorentz/caddy-run.svg?branch=master)](https://travis-ci.org/lucaslorentz/caddy-run)
+# caddy-supervisor [![Build Status](https://travis-ci.org/lucaslorentz/caddy-supervisor.svg?branch=master)](https://travis-ci.org/lucaslorentz/caddy-supervisor)
 
 ## Introduction
-This plugin enables caddy to run executables as background processes.
+This plugin enables caddy to run and supervise background processes.
 
 ## How it works
-For every **run** caddy directive a command is executed in background and killed when caddy stops.
+For every **supervisor** caddy directive a command is executed in background and killed when caddy stops.
 
-## Caddyfile syntax
-The simplest configuration requires only to specify the command and args:
+You can use **supervisor** plugin as an http directive or as a server type.
+
+## Supervisor http directive
+You can activate a supervisor directly from your web caddyfile using:
 ```
-run command arg1 arg2 arg3
+supervisor command arg1 arg2 arg3
```

Comment:
I do like `supervise` for httpdirective, but I think having the servertype and the directive with same name is a plus.